docs/index.md

```diff
@@ -21,7 +21,7 @@ title: DnsControl
          <strong><a href="migrating">migrate</a></strong>
          an existing one. Read the
          <strong><a href="js">language spec</a></strong>
-         for more info.
+         for more info. You can also <strong><a href="toc">view a list of all topics</a></strong>.
```

Comment:
I was thinking about deleting the toc.md file since it duplicates whats on index.md.

Reply:
@captncraig Thoughts?

Reply:
Well, I had quite a hard time sorting and finding he links on the homepage. It's better to have a page for use as a list of all pages to make it easier to find something if you know what you are looking for.